extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson-common/deployment/src/main/java/io/quarkus/resteasy/reactive/jackson/deployment/processor/JacksonFeatureBuildItem.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.resteasy.reactive.jackson.deployment.processor;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Contains the special Jackson features required by the application
+ */
+public final class JacksonFeatureBuildItem extends MultiBuildItem {
+
+    private final Feature feature;
```

Comment:
We aren't making use of this now, but it's there to future proof the build item in case we want to make the feature enablement really granular